.mailmap

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# This file maps author and committer email addresses to canonical names and email
+# addresses for display in the output of `git log` and `git blame`.
+# Format:
+# Canonical Name <canonical@address.dns> <actual@address.dns>

```

Comment:
Is `address.dns` actually reserved? I think this should be either `example.com` or anything ending in `.example`.

Reply:
"Contoso" would be funny. But yes, I think `example.com` makes sense.
```suggestion
# Canonical Name <canonical@example.com> <actual@example.com>
```

Reply:
Arguably `contoso.example` and `fabrikam.example` would be even better by depicting that the domain can change, but this is of course fine!